cvat/apps/engine/cache.py

```diff
@@ -91,7 +91,9 @@ def enqueue_create_chunk_job(
         with get_rq_lock_for_job(queue, rq_job_id, blocking_timeout=blocking_timeout):
             rq_job = queue.fetch_job(rq_job_id)
 
-            if not rq_job:
+            # Need to enqueue job in case of chunk been deleted but rq job is still exists
+            # This may happens in case of job with honey pots
+            if not rq_job or rq_job.get_status() == "finished":
```

Comment:
1. Maybe use `refresh=False` for better performance (you hold the lock anyway)
2. Should we check for other final statuses (`failed`, ...)?

Reply:
Added `refresh=False`, also added failed status